plugin/trino-bigquery/src/test/resources/import-tpch-to-bigquery.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
These files are not used by unit tests, so they should not be under `src/test`. Instead I would put then into `plugin/trinog-bigquery/bin`